core/domain/feedback_services.py

```diff
@@ -79,7 +78,12 @@ def create_thread(
         exploration_id, state_name, original_author_id, subject, text, False)
 
 
+def get_exp_id_thread_id_list(full_thread_id):
```

Comment:
Actually, better still: split this into two methods: get_exp_id and get_thread_id. (Or get_exp_id_from_full_thread_id and get_thread_id_from_full_thread_id.)